contrib/build_executable.jl

```diff
@@ -109,37 +108,62 @@ function build_executable(exename, script_file, targetdir=nothing; force=false)
     run(`$(julia) $(build_sysimg) $(sys.buildfile) native $(userimgjl) --force`)
     println()
 
+    gcc = find_system_gcc()
+    # This argument is needed for the gcc, see issue #9973
+    win_arg = @windows ? "-D_WIN32_WINNT=0x0600" : ""
+    @windowsxp_only win_arg = "-D_WINVER=0x0502"
+    incs = get_includes()
+    @windows_only begin
+        if contains(gcc, "WinRPM")
+            # This should not bee necessary, it is done due to WinRPM's gcc's
+            # include paths is not correct see WinRPM.jl issue #38
+            old_path = ENV["path"]
```

Comment:
Don't modify path. Make a copy of `ENV`, something like `ENV2 = copy(ENV)`. Then modify `ENV2["PATH"]` (please refer to that variable in all caps) and execute commands with ```setenv(`gcc blah`, ENV2)```